IDLE_DWELL/Find_Idle_Dwells.py

```diff
@@ -0,0 +1,256 @@
+         if sim_acis_commands[index]["command_type"] == "SIMTRANS":
+            # If was_an_obs is False, set the previous FP instrument to the
+            # present FP instrument. When there was a previous observation,
+            # the processing for the AA00 command does this.
+            if state_dict["was_an_obs"] == False:
+                state_dict["previous_fp_instrument"] = state_dict["present_fp_instrument"]
+            # Now set the present FP instrument to this SIMTRANS command value
+            state_dict["present_fp_instrument"] = BFCI.Get_Instrument(sim_acis_commands[index]["tlmsid_string"])
+
+        # Is this a Start Science Command?
+         if ("XTZ0000005" in  sim_acis_commands[index]["tlmsid_string"]) or \
+           ("XCZ0000005" in  sim_acis_commands[index]["tlmsid_string"]):
+            # If yes, you have enough information to calculate the Dwell Time
+            dwell_stop_date = sim_acis_commands[index]["date"].lstrip().rstrip()
+            dwell_stop_time = sim_acis_commands[index]["time"]
+    
+            # Calculate the delta t
+            dwell_time_secs, dwell_time_mins, dwell_time_hours = cd.Calc_Delta_Time(dwell_start_date, dwell_stop_date)
+            
+            # If the instrument is ACIS-I or ACIS-S or we are in the radzone
+            # executing an ECS measurement, and the dwell time is 1.5 hours or more
+            # then create an entry for the long_dwell_list.
+            if ( (state_dict["present_fp_instrument"] == "ACIS-I") or \
+                 (state_dict["present_fp_instrument"] == "ACIS-S") or \
+                 (state_dict["radmon_status"] == "RADMON_DIS") ) and \
+                 (dwell_time_secs >= long_dwell_cutoff):
+                dwell_string = " ".join((">>> Warning - Long Dwell:", state_dict["present_fp_instrument"], "in the F.P.",  state_dict["radmon_status"], "Dwell time: ", str(round(dwell_time_secs,1)), "seconds"))
+
+                # Print the warning out to STDOUT so that the load reviewer sees it
+                # and it gets logged in the log file we create when running lr
+                print( dwell_string)
+                
+                # Append the dwell stop date and time, and the dwell string to
+                # the list of dwell strings.  This will cause the information line
+                # to appear just before the Start Science
+                long_dwell_list.append([dwell_stop_date, dwell_stop_time, dwell_string])
+                
+            # Set the value of was_an_obs to True
+            state_dict["was_an_obs"] = True
+            
+            # Set AAOO flag to False so that we will process the first AA00
+            # command that comes after this Start Science when we get to it
+            state_dict["first_AA00_flag"] = False
+ 
+         # Check for a Stop Science: AA00000000
+         if (stop_science_command in sim_acis_commands[index]["tlmsid_string"]) and \
+           (state_dict["first_AA00_flag"] == False):
```

Comment:
same as above: `not state_dict["first_AA00_flag"]`

Reply:
Same answer as above.